pyVSR/avletters/import_data.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+#this is not guarded against whitespace!
+src_dir=/data/corpora/audiovisual/avletters
+#import audio data as is with sub directories
+mkdir data
+ln -s $src_dir/Audio data/
+#import video data with name change, dropping "-lips" from "xxx-lips.mat"
+mkdir data/Lips
+for f in $src_dir/Lips/*.mat; do ln -s $f data/Lips/$(basename ${f/-lips/}); done
+#make labels from the file name, this is just the first letter
+mkdir data/Label
+for f in data/Audio/mfcc/Clean/*.mfcc; do name=$(basename $f ".mfcc"); echo $name ${name/[0-9]_*} > data/Label/$name.mlf; done 
```

Comment:
The bash scripts wouldn't play nice on Windows platforms. This one seems easy to port.

Reply:
probably just as fast to put it in a python script that runs anywhere

Reply:
weird, should be "data/Label/$name.lab"